.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["es2015", "react"],
+  "plugins": [["react-templates", {"ext": ".rt.html", "modules": "commonjs"}]]
+}
```

Comment:
I need react-templates in order to run jasmine test of base_component and route_link.